common/lib/capa/capa/tests/test_input_templates.py

```diff
@@ -930,7 +921,7 @@ def test_status(self):
             self.assert_has_xpath(xml, xpath, self.context)
 
             # Expect a <p> with the status
```

Comment:
Update the comment. Or given that you kept the surrounding p with class "drag-and-drop--status", do you want to make the xpath include that HTML as well?